python/hsfs/core/feature_group_api.py

```diff
@@ -302,6 +302,43 @@ def get_parent_feature_groups(self, feature_group_instance):
             explicit_provenance.Links.Direction.UPSTREAM,
             explicit_provenance.Links.Type.FEATURE_GROUP,
         )
+    
+    def get_parent_storage_connector(self, feature_group_instance):
```

Comment:
Jim wanted short method names.
Parent/child are mentioned in the API when you really have to. A fg can have a parent and a child fg.
But a fg can only have a parent storage connector.
I think can reduce this to `get_storage_connector`

Reply:
OK, but now thinking about it are these hsfs changes (`get_storage_connector`) even necessary only on demand (external) fg will have storage connector and when you get it you get the associated storage connector as well.